knative-operator/Dockerfile

```diff
@@ -15,7 +15,6 @@ FROM $GO_RUNTIME
 
 ARG VERSION=
 
-RUN microdnf install tzdata 
```

Comment:
What was the goal of this before?

Reply:
I think, it's added for some other component not S-O needs.

Reply:
https://github.com/openshift-knative/hack/pull/250 ahh so it's part of ubi8-minimal already.